packages/coze-taro/src/api.ts

```
@@ -0,0 +1,47 @@
+import {
+  CozeAPI as InnerCozeAPI,
+  type ClientOptions as InnerClientOptions,
+} from '@coze/api';
+
+import { sharedMixins } from './mixins/shared';
+import { platformMixins } from './mixins/platform';
+
+export interface ClientOptions extends InnerClientOptions {
+  onBeforeAPICall?: (
+    options: unknown,
+  ) => (string | void) | Promise<string | void>;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)